src/AppBundle/Entity/Adherent.php

```diff
@@ -331,4 +333,33 @@ public function getLastLoggedAt()
 
         return $this->lastLoggedAt;
     }
+
+    /**
+     * Joins a committee as a host person.
+     *
+     * @param Committee $committee
+     *
+     * @return CommitteeMembership
+     */
+    public function joinCommitteeAsHost(Committee $committee)
```

Comment:
Hum why not but it doesn't mean the same thing. I want to reflect that you're joining the group not managing it as this point.